test/core/events.js

```diff
@@ -77,6 +77,21 @@ describe('Core htmx Events', function() {
     }
   })
 
+  it('events accept an options argument', function() {
```

Comment:
Would you mind adding a test case for the boolean `useCapture` counterpart to `options` as well? Just to make sure we support both cases and prevent any future regression on this

Reply:
As per https://github.com/bigskysoftware/htmx/pull/2836#issuecomment-2305544011, I don’t think there’s any value in testing each value that the `addEventListener()` method accepts. I’ll mark this as resolved and we can continue discussing below.